frontend/pages/dashboard/profile.tsx

```diff
@@ -0,0 +1,37 @@
+// @ts-nocheck
+import { NextPage } from 'next'
+import { withApollo } from '../../lib/apollo'
+import DashboardLayout from '../../components/Layouts/DashboardLayout'
+import Profile from '../../components/Dashboard/Profile'
+import { usePostsQuery } from '../../generated/graphql'
+
+interface InitialProps {
+  namespacesRequired: string[]
+}
+
+const ProfilePage: NextPage<InitialProps> = () => {
+  const { loading, data, error } = usePostsQuery({
+    variables: {
+      // TODO(remove hardcoded value once currentUser is available on the application state)
+      authorId: 1,
```

Comment:
I thought this TODO would have been taken care of when you merged in `<AuthGate>` and updated accordingly no?